src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -233,7 +241,7 @@ export const validateAddress = (
         type: ADDRESS_VALIDATION_CONFIRM,
         addressFromUser: userEnteredAddress.address, // need to use the address with iso3 code added to it
         addressValidationType: fieldName,
-        selectedAddress: suggestedAddresses[0], // always select the first address as the default
+        selectedAddress: confirmedSuggestions[0], // always select the first address as the default
```

Comment:
By default we need to select the first of the _confirmed_ addresses we got from the API. The bug was that we were passing the first overall suggestion, which in some cases could have been a bad address. And if the user just accepted that default selection we'd send a bad address to the update endpoint. Also, this led to an issue where the UI would show a selected address that didn't match the data in Redux. 😱